### ISSUE TITLE
Daily build: build pane has display issues

### ISSUE THREAD

**Issue by florisvdh** [rstudio/rstudio#7725]
### System details

```
RStudio Edition : Desktop
RStudio Version : 1.4.774
OS Version : Linux Mint 20 (~Ubuntu 20.04)
R Version : 4.0.2
```

### Steps to reproduce the problem

1. open a bookdown project (with index.Rmd)
1. push the build button
1. push the build button again (2nd build)

### Describe the problem in detail

- First build (step 2): font is too small

![Schermafdruk van 2020-09-03 19-08-49](https://user-images.githubusercontent.com/19164640/92146097-7bfd1c80-ee19-11ea-977c-6e91108621d4.png)

- Second build (step 3) and all later builds: build pane becomes white, nothing is displayed during build, hence no errors can be seen

![Schermafdruk van 2020-09-03 19-09-55](https://user-images.githubusercontent.com/19164640/92146105-83242a80-ee19-11ea-9e39-acd8fc1422db.png)

Note that these issues **did not occur in `1.4.751`**.

### Describe the behavior you expected

Normal building pane font and display is expected.

- [X] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [X] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [X] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by kevinushey** [comment]
I believe this is fallout from https://github.com/rstudio/rstudio/pull/7568. In the meantime, you can work around this by disabling the 'Limit console display to a subset of total content' preference here:

<img width="598" alt="Screen Shot 2020-09-03 at 12 04 05 PM" src="https://user-images.githubusercontent.com/1976582/92156300-a16c3500-eddd-11ea-97b4-18549f90a3b5.png">

**Comment by florisvdh** [comment]
Thanks for the feedback @kevinushey .

**Comment by dfalty** [comment]
It looks like this is fixed for me on version 1.4.873. @florisvdh please reopen if it is not fixed for you.

**Comment by florisvdh** [comment]
I confirm that this is solved (observed in 1.4.875). :+1:
